includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,21 +1,185 @@
 <?php
 
 class DesignSystemApiController extends WikiaApiController {
+	private $wikiId;
+	private $lang;
+
+	private $data = [
+		'brand-logo' => [
+			'type' => 'brand-logo',
+			'href' => '#',
+			'elements' => [
+				[
+					'type' => 'line-image',
+					'image' => 'company/logo-fandom',
+					'title' => [
+						'type' => 'translatable-text',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "translatable-text" 16 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)